Public/Src/Sandbox/Windows/DetoursServices/FilesCheckedForAccess.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "FilesCheckedForAccess.h"
+#include "string.h"
+
+FilesCheckedForAccess FilesCheckedForAccess::Create()
+{
+    return FilesCheckedForAccess();
+}

```

Comment:
Do we need `Create()` ?

Reply:
Good catch, we don't. Leftover from another impl

---
In reply to: [299195215](https://github.com/microsoft/BuildXL/pull/536#discussion_r299195215) [](ancestors = 299195215)